community/values/src/main/java/org/neo4j/values/virtual/ArrayHelpers.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.values.virtual;
+
+import org.neo4j.values.AnyValue;
+import org.neo4j.values.Values;
+
+/**
+ * This class is way too similar to org.neo4j.collection.primitive.PrimitiveArrays.
+ *
+ * Should we introduce dependency on primitive collections?
+ */
+class ArrayHelpers
```

Comment:
we usually have static helpers being final with a `private` constructor so that they cannot be instantiated